.circleci/config.yml

```diff
@@ -137,33 +137,33 @@ package_steps: &package_steps
 jobs:
   main:
     docker:
-      - image: circleci/node:14
+      - image: circleci/node:16
 
     <<: *main_steps
 
-  main@node-16:
+  main@node-17:
     docker:
-      - image: circleci/node:16
+      - image: circleci/node:17
```

Comment:
I don't think we should actually aim to deploy on Node 17 at any point. There won't be a node 18 build available until April 2022 but I reckon me might as well keep a 'next' build in play and then switch it to 18 next year.